rudder-relay/debian/postinst

```diff
@@ -51,6 +51,11 @@ case "$1" in
         mv /opt/rudder/etc/rudder-pkg/rudder-pkg.conf.distrib /opt/rudder/etc/rudder-pkg/rudder-pkg.conf
       fi
 
+      # sometimes (ubuntu22 ?) divert prevents the file to be present
+      if [ ! -f /opt/rudder/etc/rudder-networks-24.conf ]; then
+        touch /opt/rudder/etc/rudder-networks-24.conf
```

Comment:
shouldn't we move the .distrib file?

Reply:
we shouldn't need to, but we can, this way the directory would be cleaned from .distrib files